filebeat/files/filebeat.jinja

```diff
@@ -1,73 +1,27 @@
-{%- set filebeat = salt['pillar.get']('filebeat', {}) %}
+# Example configuration file
```

Comment:
this file appears not to be used any more, and probably should be deleted.

put examples of config in the pillar.example file, not here.